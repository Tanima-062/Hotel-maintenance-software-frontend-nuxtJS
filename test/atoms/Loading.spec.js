import { shallowMount } from '@vue/test-utils'
import Loading from '@/components/atoms/Loading.vue'

describe('Atoms Loading Check', () => {
  test('Loading Test', () => {
    const wrapper = shallowMount(Loading)

    // check if it is a vue element
    expect(wrapper.vm).toBeTruthy()

    // check Snapshot
    expect(wrapper.element).toMatchSnapshot()
  })
})
