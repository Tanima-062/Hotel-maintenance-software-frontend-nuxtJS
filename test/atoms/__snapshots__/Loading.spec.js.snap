// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms Loading Check Loading Test 1`] = `
<div
  class="spinner"
>
  <div
    class="bounce1"
  />
   
  <div
    class="bounce2"
  />
   
  <div
    class="bounce3"
  />
</div>
`;
